src/features/content-explorer/item-list/ItemListName.js

```diff
@@ -2,15 +2,15 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import PlainButton from '../../../components/plain-button';
+import { TYPE_FOLDER } from '../../../constants';
```

Comment:
could we move this out of this section? we don't follow this too well but it'd be good to group similar imports together. e.g. grouping them like:
```
// base/root imports
// external libs
// internal libs
// components
// types
// constants, messages
// styles
```
some of the sections could be swapped around but that's how I've generally seen them

Reply:
Yup. I think most of these were automatic imports, there's probably a way to configure it somehow 🤔